src/AWS.DistributedCacheProvider/AWS.DistributedCacheProvider.csproj

```diff
@@ -35,7 +35,7 @@
 		<PackageReference Include="Microsoft.Extensions.Caching.Abstractions" Version="6.0.0" />
 		<PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="6.0.0" />
 		<PackageReference Include="Microsoft.Extensions.Options" Version="6.0.0" />
-		<PackageReference Include="AWSSDK.DynamoDBv2" Version="3.7.103.9" />
+		<PackageReference Include="AWSSDK.DynamoDBv2" Version="3.7.302.15" />
```

Comment:
Could you update THIRD_PARTY_LICENSES too (and the internal tool, if we're using it for this repo)?

Reply:
Done